.github/workflows/python.yaml

```diff
@@ -59,3 +59,9 @@ jobs:
       run: echo "${DOCKER_PASSWORD}" | docker login -u ${DOCKER_USERNAME} --password-stdin ${DOCKER_REGISTRY_URL}
     - name: Push image
       run: docker push docker-registry.siecobywatelska.pl/small_eod:latest
+    - name: Trigger SDK regenerate
+      uses: peter-evans/repository-dispatch@v1
+      with:
+        token: ${{ secrets.SDK_REPO_TOKEN }}
```

Comment:
Czy to @ad-m używa twojego osobistego klucza?

Reply:
tfu, tokenu?

Reply:
Pytam, bo:

![Zrzut ekranu z 2020-06-26 11-39-46](https://user-images.githubusercontent.com/1165138/85843672-c91d9c00-b7a1-11ea-92a4-baac93c8665d.png)

Reply:
Tak, z ograniczonymi uprawnieniami. Możliwe, że za jakiś czas stworzę konto użytkownika integracyjne, ale pierw chce napisać do supportu GitHub czy planują jakoś efektywniej rozwiązać kwestie tych tokenów w rozsądnym oknie czasowym.

Reply:
My w CfP póki co mamy takie konto @CodeForPoznanBot do podobnych rzeczy (chociażby na tym koncie wisi nasz Appveyor).